lib/ASRouterTargeting.jsm

```diff
@@ -176,6 +179,12 @@ const TargetingGetters = {
   },
   get totalBookmarksCount() {
     return TotalBookmarksCountCache.getTotalBookmarksCount;
+  },
+  get firefoxVersion() {
+    return parseInt(AppConstants.MOZ_APP_VERSION, 10);
+  },
+  get geoCountry() {
```

Comment:
maybe we should call this `region` to be more consistent with serverside filtering